h/schemas/annotation.py

```diff
@@ -92,7 +92,10 @@ class AnnotationSchema(JSONSchema):
                 "type": "array",
                 "items": {
                     "type": "object",
-                    "properties": {"selector": copy.deepcopy(SELECTOR_SCHEMA)},
+                    "properties": {
+                        "description": {"type": "string", "maxLength": 250},
```

Comment:
There has to be some max length on the `target.description` field but I don't have an opinion about what that max length should be. Arbitrarily chose 250 chars here but happy to change it to something else.

Reply:
"Tweet length" is what I had in mind. We won't really have room for more than that in the UI.